src/components/project/project.rules.ts

```diff
@@ -1019,7 +1019,7 @@ export class ProjectRules {
       .match([
         node('node', 'Project', { id }),
         relation('out', '', 'step', INACTIVE),
-        node('prop'),
+        node('prop', 'Deleted_ProjectStep'),
```

Comment:
This is too avoid step changes inside of changeset right @daniel-leod?

@zkhin what do you think about this change? does it make sense?

Reply:
I think no. If we need to get the previous step in the changeset, I think we will need to refactor `getPreviousSteps` similar to `getCurrentStep`.

Reply:
I think we are trying to avoid the step set in a changeset. But honestly this whole workflow needs to be updated to work in tandem with changesets